doc/examples/data/plot_3d.py

```diff
@@ -0,0 +1,25 @@
+"""
+==========================================
+datasets with 3 or more spatial dimensions
+==========================================
+
+Most scikit-image functions are compatible with 3-D datasets, ie images with
+3 spatial dimensions (as opposed to 2-D multichannel arrays, which also have
+three axes). func:`skimage.data.cells3d` returns a 3D fluorescence microscopy
+image of cells. The returned data is a 3D multichannel array with dimensions
```

Comment:
```suggestion
image of cells. The returned dataset is a 3D multichannel image with dimensions
```
Alternatively, it's a 4D array (like, `img.ndim`)... right?